index.js

```diff
@@ -7,8 +7,7 @@
 const os = require('os')
 const debug = require('debug')('speaker')
 const binding = require('bindings')('binding')
-const bufferAlloc = require('buffer-alloc')
-const Writable = require('readable-stream/writable')
+const { Writable } = require('readable-stream')
```

Comment:
Considering we're dropping support for Node < 8, I think the `buffer-alloc` and `readable-streams` modules can be dumped in favor of their Node core built-in equivalents.